### PR TITLE
save fixed input data in the model

### PR DESCRIPTION
**Pull request by elisehuard**
models can have data sources which are not necessarily meant to be modified by the user.  So it's necessary to include them as model parameters, in a way.

### CODE REVIEW

**Comment by acron0** [comment]
I know this PR is "in progress" and likely to change but is there a reason the "parameter data" is not amalgamated with "input data"? As far as I can see the only difference is that one can change and one can't - i.e. a read-only data input.

File: src/witan/app/model.clj [hunk 1]
```diff
@@ -19,20 +20,23 @@
...
       (assoc :version-id version_id
              :model-id model_id
              :created (util/java-Date-to-ISO-Date-Time created)
-             :input-data (mapv #(-> (hash-map :category %1)
-                                    (cond-> (get input_data_defaults %1) (assoc :default (data/Data-> (get input_data_defaults %1))))) input_data)
-             :output-data (mapv #(hash-map :category %1) output_data))))
+             :fixed-input-data (mapv #(-> (hash-map :category %1)
+                                    (cond-> (get input_data_defaults %1)(assoc :default (data/Data-> (get input_data_defaults %1))))) input_data)
+             :output-data (mapv #(hash-map :category %1) output_data)
+             :fixed-input-data (mapv (fn [[category data]] (hash-map :category category :data (data/Data-> data))) fixed_input_data))))
```
Comment (acron0) [inline src/witan/app/model.clj]: `:fixed-input-data` appears twice

Comment (elisehuard) [inline src/witan/app/model.clj]: doh, was over-zealous, that's input-data

**Comment by acron0** [comment]
lgtm :shipit: - my only concern is that sw1nn might be deep in this code as well

**Pull request merged**